diffkemp/simpll/ModuleAnalysis.cpp

```diff
@@ -174,7 +177,8 @@ void simplifyModulesDiff(Config &config, OverallResult &Result) {
                  mam.getResult<CalledFunctionsAnalysis>(*config.First,
                                                         config.FirstFun),
                  mam.getResult<CalledFunctionsAnalysis>(*config.Second,
-                                                        config.SecondFun));
+                                                        config.SecondFun),
+                 config);
```

Comment:
In the end, I decided to pass just the built-in pattern configuration structure instead of the whole config - it's explicit which `Config` fields the `DebugInfo` constructor actually uses.